Client/Ecosia/UI/NTP/OnboardingCard/NTPOnboardingCardCellViewModel.swift

```diff
@@ -0,0 +1,77 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Shared
+import Core
+import Common
+
+protocol NTPOnboardingCardCellDelegate: AnyObject {
+    func onboardingCardClick()
+    func onboardingCardDismiss()
+}
+
+final class NTPOnboardingCardCellViewModel {
+    weak var delegate: NTPOnboardingCardCellDelegate?
+    var theme: Theme
+    
+    init(delegate: NTPOnboardingCardCellDelegate? = nil, theme: Theme) {
+        self.delegate = delegate
+        self.theme = theme
+    }
+}
+
+extension NTPOnboardingCardCellViewModel: HomepageViewModelProtocol {
+    func setTheme(theme: Theme) {
+        self.theme = theme
+    }
+
+    var sectionType: HomepageSectionType {
+        return .onboardingCard
+    }
+
+    var headerViewModel: LabelButtonHeaderViewModel {
+        return .emptyHeader
+    }
+
+    func section(for traitCollection: UITraitCollection, size: CGSize) -> NSCollectionLayoutSection {
+        let itemSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0),
+                                              heightDimension: .fractionalHeight(1.0))
+        let item = NSCollectionLayoutItem(layoutSize: itemSize)
+
+        let groupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0),
+                                               heightDimension: .estimated(128)) // TODO: Make cell automatically size it's height
```

Comment:
Struggled a bit to make it resize it's height automatically, even setting up all constraints. Must be missing something but did not have the proper time to debug.

Reply:
Solved ✅ . Adding this comment to track it 👍 . The issue was the right `.estimated()`dimension to assign to both the item and the group.